docs/source/notes/numerical_accuracy.rst

```diff
@@ -51,6 +51,40 @@ datatype. E.g.:
     a.norm() # produces tensor(inf)
     a.double().norm() # produces tensor(1.4142e+20, dtype=torch.float64), representable in fp32
 
+.. _Linear Algebra Stability:
+
+Linear algebra (``torch.linalg``)
+---------------------------------
+
+Non-finite values
+^^^^^^^^^^^^^^^^^
+
+The external libraries (backends) that ``torch.linalg`` uses provide no guarantees on their behaviour
+when the inputs have non-finite values like ``inf`` or ``NaN``. As such, neither does PyTorch.
+The operations may return a tensor with non-finite values, or raise an exception, or even segfault.
+
+Consider using :func:`torch.isfinite` before calling these functions if your inputs may contain these values.
+
+Extremal values
+^^^^^^^^^^^^^^^
+
+What's considered `Extremal Values`_ in the context of ``torch.linalg`` is a bit more subtle than in the rest of PyTorch.
+
+:ref:`linalg solvers` and :ref:`linalg inverses` assume that the input matrix ``A`` is invertible. If it is close to
+being, non-invertible (for example, a matrix that a has a very small singular value), then these algorithms may silently return
+incorrect results. These matrices are said to be `ill-conditioned <https://nhigham.com/2020/03/19/what-is-a-condition-number/>`_.
+If provided with ill-conditioned inputs, the results of these functions may vary when using the same input on different devices
+or when using different backends via the keyword ``driver``.
+
+Similarly, spectral operations like ``svd``, ``eig``, and ``eigh`` may return incorrect results (and their gradients may be infinite)
```

Comment:
What about a separate heading: "Matrices with close eigenvalues"?

Reply:
Do we have a suggestion for what the user can do in this case?

Reply:
In the end, rather than making these their own heading, I made sure that the first words in the sentence describe the important set of operations that are described in the paragraph. Making these into their own heading did not look great when rendered.